test/integration/targets/ecs_cluster/playbooks/roles/ecs_cluster/tasks/main.yml

```diff
@@ -754,10 +759,69 @@
         <<: *aws_connection_info
       register: fargate_run_task_output
 
-    - name: assert that public IP assignment is enabled
-      assert:
-        that:
-          - 'ecs_fargate_service_network_with_awsvpc.service.networkConfiguration.awsvpcConfiguration.assignPublicIp == "ENABLED"'
+      # aws cli not installed in docker container; make sure it's installed.
+    - name: install awscli
+      pip:
+        state: present
+        name: awscli
+
+    - name: disable taskLongArnFormat
```

Comment:
This can be expressed far more healthily (i.e. you can see what has gone wrong because you don't have to use no_log) as:

```
      command: aws ecs put-account-setting --name taskLongArnFormat --value disabled
      environment:
        AWS_ACCESS_KEY_ID: "{{ aws_access_key }}"
        AWS_SECRET_ACCESS_KEY: "{{ aws_secret_key }}"
        AWS_SESSION_TOKEN: "{{ security_token }}"
        AWS_DEFAULT_REGION: "{{ aws_region }}"
```

Reply:
I'll probably just push up my fix for this shortly, so don't worry too much about it :)